File: Examples/test-suite/matlab/input_runme.m
f = input.Foo();
if (f.foo(2) ~= 4)
  error
end

try
  a=f.foo();
  error
catch
end

if (~strcmp(sfoo('Hello'),'Hello world'))
  error
end

try
  a=sfoo();
  error
catch
end

File: Examples/test-suite/javascript/input_runme.js
var input = require("input");

var f = new input.Foo();
if (/* await */(f.foo(2)) != 4) {
    throw new Error;
}

if (/* await */(f.foo(null)) != null) {
    throw new Error;
}

if (/* await */(f.foo()) != null) {
    throw new Error;
}

if (/* await */(input.sfoo("Hello")) != "Hello world") {
    throw new Error;
}

if (/* await */(input.sfoo()) != null) {
    throw new Error;
}
